docs/HowToFAQ/check-errors-a

```diff
@@ -34,7 +34,7 @@ done
 ## ErrorIds from Add
 
 for f in $errfiles ; do
-  echo ENUM FILE $f
+  echo ENUM FILE ADD $f
```

Comment:
I can't figure out what these symbols are for and hence what `ADD` is doing.

Reply:
Just debugging output. I'll add comments in a future PR.